src/sentry/templates/sentry/js-sdk-loader.ts

```diff
@@ -126,6 +114,32 @@ declare const __LOADER__IS_LAZY__: any;
     _currentScriptTag.parentNode!.insertBefore(_newScriptTag, _currentScriptTag);
   }
 
+  // We want to ensure to only add default integrations if they haven't been added by the user.
+  function setupDefaultIntegrations(config: any, SDK: any) {
+    const integrations: {name: string}[] = config.integrations || [];
```

Comment:
```suggestion
    const integrations: {name: string}[] = config.integrations ?? [];
```

Reply:
This transpiles into a slightly more verbose form, which we generally try to avoid for ES5 code, as it adds a few - in this case unnecessary - bytes. In this case, I think `||` is fine. (Working on the JS SDK burns this into your brain xD)